### PR TITLE
tests: using labeler action to add automatically a label to run nested tests

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The idea is to add the "Run nested" label automatically in case either
the nested lib or any nested test changed.

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
This is documented here: 
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target
and 
https://github.com/actions/labeler

The workflow will work in the project but once it is merged.

It was tested here: https://github.com/sergiocazzolato/snappy-qa-jobs/pull/5

File: .github/workflows/labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@main
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (bboozzoo) [inline .github/workflows/labeler.yaml:11]: missing newline at the end

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by mvo5 — APPROVED** [review]
Nice feature, thank you!

**Pull request merged**